test/datasets_utils.py

```diff
@@ -44,7 +44,7 @@ class UsageError(Exception):
 
 
 class LazyImporter:
-    r"""Lazy importer for additional dependicies.
+    r"""Lazy importer for additional dependencies.
```

Comment:
Good catch! Could you revert this here and send a follow-up PR in order to keep the scope small?